src/plonk.rs

```diff
@@ -14,7 +14,7 @@ use crate::transcript::ChallengeScalar;
 mod circuit;
 mod keygen;
 mod lookup;
-mod permutation;
+pub(crate) mod permutation;
```

Comment:
Ideally we wouldn't need to do this (the permutation module here should use a shared implementation of @daira and @defuse's permutation algorithm) but this is fine for now.